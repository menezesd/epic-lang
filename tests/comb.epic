func main(){
    print(C(5, 3));
    print(C(7, 2));
    print(C(14, 6));
    print F(8);
}
func C(n, k){return F(n)/(F(n-k)*F(k));}
func F(n){
    res=1;
    for i in 1..n+1 do{
        res=res*i;
    }
    return res;
}
