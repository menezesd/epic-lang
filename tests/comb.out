10
21
3003
40320
